RecoTracker/FinalTrackSelectors/plugins/TrackCutClassifier.cc

```diff
@@ -36,13 +38,36 @@ namespace {
     
   inline float chi2n(reco::Track const & tk) { return tk.normalizedChi2();}
 
+  inline float relPtErr(reco::Track const & tk) {
+    return  float(tk.ptError())/float(tk.pt());
```

Comment:
sure
I was waiting for feedback
I'll do it tomorrow morning
thanks


> On 06 Jul 2016, at 22:15, David Lange <notifications@github.com> wrote:
> 
> In RecoTracker/FinalTrackSelectors/plugins/TrackCutClassifier.cc:
> 
> > @@ -36,13 +38,36 @@ namespace {
> >      
> >    inline float chi2n(reco::Track const & tk) { return tk.normalizedChi2();}
> >  
> > +  inline float relPtErr(reco::Track const & tk) {
> > +    return  float(tk.ptError())/float(tk.pt());
> @mtosi - could you just update this PR?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>